core-api/src/test/java/com/sequenceiq/distrox/api/v1/distrox/model/DistroXV1RequestTest.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.distrox.api.v1.distrox.model;
+
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.distrox.api.v1.distrox.model.instancegroup.InstanceGroupV1Request;
+
+class DistroXV1RequestTest {
+
+    @Test
+    public void getAllRecipesIfRecipeNamesAreNullTest() {
+        DistroXV1Request distroXV1Request = new DistroXV1Request();
+        InstanceGroupV1Request instanceGroupV1Request = new InstanceGroupV1Request();
+        instanceGroupV1Request.setRecipeNames(null);
+        distroXV1Request.setInstanceGroups(Collections.singleton(instanceGroupV1Request));
+        distroXV1Request.getAllRecipes();
```

Comment:
Shouldn't we verify that the result is an empty set?

Reply:
We can verify, I just wanted to create as minimal unit test as I can to test this use case :D anyway, assert is added